### PR TITLE
Improvements update

### PR DESCRIPTION
**Pull request by brandon-binary**

### CODE REVIEW

File: src/templates/static/get_started/binary_options_mt5.jsx [hunk 1]
```diff
@@ -16,6 +16,12 @@ const BinaryOptionsForMT5 = () => (
                             { id: 'how-to-trade-binary',     href: '#how-to-trade-binary',     text: it.L('How to trade binary options on MT5') },
                         ]}
                     />
+
+                    <div className='center-text gr-padding-10 client_logged_out invisible'>
+                        <a className='button' href={it.url_for('new-account')}>
+                            <span>{it.L('Open a free account')}</span>
```
Comment (negarn) [inline src/templates/static/get_started/binary_options_mt5.jsx]: as far as I see they wanted a button to open an MT5 account, not a binary account. the page needs to be updated, see the same button at the bottom of the get-stareted/metatrader. also why are we only adding to one of the sub pages of the get-started/metatrader? @ashkanx

File: src/templates/static/metatrader/types_of_accounts.jsx [hunk 2]
```diff
@@ -102,19 +102,19 @@ const TypesOfAccounts = () => (
...
-                                [{ text: <Row number={10} text={it.L('Maximum ticket size')} /> },    { text: it.L('[_1] lots', '10') },                { text: it.L('[_1] lots', '20') },                { text: it.L('[_1] lots', '30') }],
-                                [{ text: <Row number={11} text={it.L('Minimum ticket size')} /> },    { text: it.L('[_1] lots', '0.01') },              { text: it.L('[_1] lots', '0.01') },              { text: it.L('[_1] lots', '0.01') }],
-                                [{ text: <Row number={12} text={it.L('Step size')} /> },              { text: it.L('[_1] lots', '0.01') },              { text: it.L('[_1] lots', '0.01') },              { text: it.L('[_1] lots', '0.01') }],
-                                [{ text: <Row number={13} text={it.L('Cryptocurrency trading')} /> }, { text: '24/7' },                                 { text: 'N/A' },                                  { text: it.L('N/A') }],
+                                [{ text: <Row text={it.L('Account currency')} /> },       { text: it.L('USD') },                            { text: it.L('USD') },                            { text: it.L('USD/EUR') }],
```
Comment (negarn) [inline src/templates/static/metatrader/types_of_accounts.jsx]: please keep the indentation even if the number attribute has been removed, it's nice to align by `text=`

File: src/templates/static/get_started/binary_options_mt5.jsx (hunk 1 above)
Comment (ashkanx) [reply src/templates/static/get_started/binary_options_mt5.jsx]: yes, the button should be for MT5 account

Comment (negarn) [reply src/templates/static/get_started/binary_options_mt5.jsx]: @brandon-binary please add the mt5 account opening button to the bottom of the menu of all get started/metatrader sub pages then. better to use a common component in that case.

File: src/templates/static/get_started/common.jsx [hunk 3]
```diff
@@ -97,3 +97,13 @@ export const MtBox = ({ text, icon_1, icon_2 }) => (
         <div className={`mt-img ${icon_2}`} />
     </div>
 );
+
+export const SideBarCTA = () => (
+    <div id='sidebar-cta' className='center-text gr-padding-10 client_logged_in invisible'>
+        <p>
+            <a className='button' href={it.url_for('user/metatrader')}>
+                <span>Create an MT5 account now.</span>
```
Comment (kellybinary) [inline src/templates/static/get_started/common.jsx]: could you wrap this sentence in `{it.L('...')}` to add this to translation too?

File: src/templates/static/get_started/common.jsx [hunk 4]
```diff
@@ -97,3 +97,13 @@ export const MtBox = ({ text, icon_1, icon_2 }) => (
         <div className={`mt-img ${icon_2}`} />
     </div>
 );
+
+export const SideBarCTA = () => (
+    <div id='sidebar-cta' className='center-text gr-padding-10 client_logged_in invisible'>
```
[the remark this answers is not in the document]
Comment (brandon-binary) [reply src/templates/static/get_started/common.jsx]: @negarn actually we aren't, I added that earlier, I confused it for the other create MT5 button

**Pull request merged**